idaes/gas_solid_contactors/unit_models/fixed_bed_0D.py

```diff
@@ -0,0 +1,361 @@
+                    return (
+                            b.solids[t].temperature ==
+                            b.solids[0].temperature)
+
+    def initialize(blk, state_args=None, outlvl=idaeslog.NOTSET,
+                   solver=None, optarg=None):
+        """
+        This is a general purpose initialization routine for simple unit
+        models. This method assumes a single ControlVolume block called
+        controlVolume, and first initializes this and then attempts to solve
+        the entire unit.
+        More complex models should overload this method with their own
+        initialization routines.
+
+        Keyword Arguments:
+            state_args : a dict of arguments to be passed to the property
+                           package(s) to provide an initial state for
+                           initialization (see documentation of the specific
+                           property package) (default = {}).
+            outlvl : sets output level of initialization routine
+                 * 0 = Use default idaes.init logger setting
+                 * 1 = Maximum output
+                 * 2 = Include solver output
+                 * 3 = Return solver state for each step in subroutines
+                 * 4 = Return solver state for each step in routine
+                 * 5 = Final initialization status and exceptions
+                 * 6 = No output
+            optarg : solver options dictionary object (default={'tol': 1e-6})
+            solver : str indicating which solver to use during
+                     initialization (default = 'ipopt')
+
+        Returns:
+            None
+        """
+
+        # Set solver options
+        init_log = idaeslog.getInitLogger(blk.name, outlvl)
+        opt = get_solver(solver, optarg)  # create solver
+        opt.options = optarg
+
+        # ---------------------------------------------------------------------
+        # Initialize control volume block
+        init_log.info('Initialize Thermophysical Properties')
+
+        # Initialize gas_phase block
+        gas_phase_flags = blk.gas.initialize(
+                                outlvl=outlvl,
+                                optarg=optarg,
+                                solver=solver)
+
+        # Initialize solid_phase properties block
+        solid_phase_flags = blk.solids.initialize(
+                                outlvl=outlvl,
+                                optarg=optarg,
+                                solver=solver)
+
+        print()
+        print('Initialize reaction properties')
+        # Initialize reactions
+        blk.reactions.initialize(outlvl=outlvl,
+                                 optarg=optarg,
+                                 solver=solver)
+
+        init_log.info_high("Initialization Step 1 Complete.")
+
+        # ---------------------------------------------------------------------
+        # Release Inlet state
+        blk.gas.release_state(gas_phase_flags, outlvl+1)
```

Comment:
The `+1` should be removed from the output level now that we are using the logger to manage output.